### PR TITLE
Add turtlebot4 release team.

### PR DESCRIPTION
**Pull request by roni-kreinin**
For releasing [turtlebot4](https://github.com/turtlebot) related packages.

### CODE REVIEW

**Comment by kscottz** [comment]
@nuclearsandwich friendly ping. We need to get this moving. Let me know if I can help out.

**Comment by kscottz** [comment]
@nuclearsandwich friendly ping. This is blocking on TB4. We can also assign someone else if you don't have the bandwidth.

**Comment by kscottz** [comment]
@methylDragon @clalancette would you have a moment to review this fairly soon?

**Comment by methylDragon** [comment]
It LGTM, but I don't think I can add myself as a reviewer.

That said, you might want to do [lexical sort ordering](https://github.com/ros2-gbp/ros2-gbp-github-org/commit/9627ca910489733317d0232f649447ef44a16aab) of the repos and team members.

**Review by methylDragon — APPROVED** [review]
LGTM!

That said, you might want to do [lexical sort ordering](https://github.com/ros2-gbp/ros2-gbp-github-org/commit/9627ca910489733317d0232f649447ef44a16aab).

File: turtlebot4.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+locals {
+  turtlebot4_team = [
+    "roni-kreinin",
+    "tonybaltovski",
+    "tfoote",
+    "kscottz",
+  ]
```
Comment (methylDragon) [inline turtlebot4.tf]:
```suggestion
  turtlebot4_team = [
    "kscottz",
    "roni-kreinin",
    "tfoote",
    "tonybaltovski",
  ]
```

File: turtlebot4.tf [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  turtlebot4_repositories = [
+    "turtlebot4-release",
+    "turtlebot4_desktop-release",
+    "turtlebot4_robot-release",
+    "turtlebot4_simulator-release",
```
Comment (methylDragon) [inline turtlebot4.tf:12]:
```suggestion
    "turtlebot4-release",
    "turtlebot4_desktop-release",
    "turtlebot4_robot-release",
    "turtlebot4_simulator-release",
```

**Review by nuclearsandwich — APPROVED** [review]
Thanks for your patience with the delayed review. I'll have this deployed shortly.

**Comment by nuclearsandwich** [comment]
These changes have been deployed. Let me know if you have any trouble with access.

**Pull request merged**